hack/machineset.sh

```diff
@@ -185,7 +185,7 @@ get_vsphere_ms() {
 
   # set golden image template name
   # TODO: read from parameter
-  template="windows-golden-images/windows-server-2004-template"
+  template="windows-golden-images/windows-server-2022-template-networkHotfix"
```

Comment:
I don't think the `-networkHotfix` suffix is needed. 

We can stick to `windows-golden-images/windows-server-2022-template` as the latest golden image to avoid code changes. Once the final WS2022 is ready, we just replace the image in vCenter. No PR is needed.

Reply:
given `...-networkHotfix` is the name of the VM created in IBM cloud environment, I will stick with this. We will need a PR anyway to move away from pulling 2004 container images for platform=none.